### PR TITLE
[CP-1131] add reducer and actions

### PR DESCRIPTION
**Pull request by wercix**
Jira: [CP-1131]

**Description**
added:
Store, Actions with getFilesAction,  Reducers,
getFilesRequest with mocked data,
UI without styling: error, loading, loaded states and list component
<details>
<summary><b>Screenshots</b></summary>
// put images here
</details>

[CP-1131]: https://appnroll.atlassian.net/browse/CP-1131?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/app/src/files-manager/files-manager.container.tsx [hunk 1]
```diff
@@ -9,6 +9,8 @@ import FilesManager from "App/files-manager/components/files-manager/files-manag
 
 const mapStateToProps = (state: RootState & ReduxRootState) => ({
   memorySpace: state.device.data?.memorySpace,
+  musicFiles: state.filesManager.files,
```
Comment (dkarski) [inline packages/app/src/files-manager/files-manager.container.tsx:12]:
```suggestion
  files: state.filesManager.files,
```

`MtpFileType` will be mark that file is music or not 😉

File: packages/app/src/files-manager/components/files-manager/files-manager.component.tsx [hunk 2]
```diff
@@ -14,9 +14,16 @@ import {
...
+import FilesStorage from "App/files-manager/components/files-storage/files-storage.component"
+import {
+  MtpFile,
+  ResultState,
+} from "App/files-manager/reducers/files-manager.interface"
 
 interface Props {
   memorySpace?: MemorySpace
+  resultState: ResultState
+  musicFiles: MtpFile[]
```
Comment (dkarski) [inline packages/app/src/files-manager/components/files-manager/files-manager.component.tsx:26]:
```suggestion
  files: MtpFile[]
```

File: packages/app/src/files-manager/components/files-manager/files-manager.component.tsx [hunk 3]
```diff
@@ -25,6 +32,8 @@ const FilesManager: FunctionComponent<Props> = ({
     full: 0,
     total: 0,
   },
+  resultState,
+  musicFiles,
```
Comment (dkarski) [inline packages/app/src/files-manager/components/files-manager/files-manager.component.tsx:36]:
```suggestion
  files,
```

File: packages/app/src/files-manager/components/files-storage/files-storage.component.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+import React from "react"
+import { FunctionComponent } from "Renderer/types/function-component.interface"
+import {
+  File as musicFile,
+  ResultState,
+} from "App/files-manager/reducers/files-manager.interface"
+
+interface Props {
+  resultState: ResultState
+  musicFiles: musicFile[]
```
Comment (dkarski) [inline packages/app/src/files-manager/components/files-storage/files-storage.component.tsx]:
```suggestion
  files: musicFile[]
```

**Review by dkarski — COMMENTED** [review]
please change `musicFiles` to `files`. `MtpFileType` will be mark that file is music or not

**Comment by dkarski** [comment]
change base branch to CP-682

**Review by dkarski — APPROVED** [review]

**Review by igorbezsmertnyi — APPROVED** [review]

**Pull request merged**